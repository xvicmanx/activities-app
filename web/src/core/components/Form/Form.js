import React from 'react';
import './Form.css';

const Form = ({ children, ...rest }) => {
  return (
    <form {...rest} className="Form">
      {children}
    </form>
  );
};

export default Form;
