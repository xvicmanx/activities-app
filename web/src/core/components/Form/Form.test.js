import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import Form from './Form';

describe('Form', () => {
  it('renders properly', () => {
    const renderer = new ShallowRenderer();
    const onClick = jest.fn()
    const result = renderer.render(<Form>Test</Form>);
    expect(result).toMatchSnapshot();
  });
});
