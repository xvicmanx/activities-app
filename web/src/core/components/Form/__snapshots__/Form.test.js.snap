// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form renders properly 1`] = `
<form
  className="Form"
>
  Test
</form>
`;
